db/migrate/20220701173654_create_active_admin_comments.rb

```diff
@@ -0,0 +1,16 @@
+class CreateActiveAdminComments < ActiveRecord::Migration[7.0]
+  def self.up
+    create_table :active_admin_comments do |t|
+      t.string :namespace
+      t.text   :body
+      t.references :resource, polymorphic: true
+      t.references :author, polymorphic: true
+      t.timestamps
+    end
+    add_index :active_admin_comments, [:namespace]
+  end
+
+  def self.down
+    drop_table :active_admin_comments
+  end
```

Comment:
Can we define the `change` method here?

Reply:
I'm not sure if active admin comments are necessary here, I think youn can disable them in the configuration file

Reply:
If we are not going to use ActiveAdmin comments, then we need to remove this migrations as well

Reply:
I agree, this migration should be remove now that you've disabled comments in the configuration file @LeandroSoria2